### DISCUSSION TITLE
ModelMultipleChoiceFilter with lookup_expr='in'

### DISCUSSION THREAD

**Discussion by emilt27** [carltongibson/django-filter#1433] [Q&A] [answered]
We had the following field definition:
```
category = django_filters.ModelMultipleChoiceFilter(
    queryset=Category.objects.all(), 
    lookup_expr='in',
)
```

it worked correctly until last django-filter release. Now it's broken. Removing `lookup_expr` solves the issue, but It would be great to have more detailed message.

**Comment by rpkilby** [comment, marked as answer]
#1054 changed this. I don't think `lookup_expr` was previously used, and by default, Q objects are OR'ed together, so the result looks like it was performing an `in` lookup. e.g., 
```python
MyModel.objects.filter(Q(field='value1') | Q(field='value2'))
```

The PR changed this so lookup expressions are accounted for, so you could queries like
```python
MyModel.objects.filter(Q(field__contains='value1') | Q(field__contains='value2'))
```

However, in your case, the `in` lookup expects a list of values, which is why you get an error. It should be safe to drop `lookup_expr='in'`
